docs/Guides/Ecosystem.md

```diff
@@ -282,6 +282,9 @@ section.
   functions.
 - [`fastify-decorators`](https://github.com/L2jLiga/fastify-decorators) Fastify
   plugin that provides the set of TypeScript decorators.
+- [`fastify-delay-request`](https://github.com/climba03003/fastify-delay-request)
+  Fastify plugin that allow to spin up the server while some request is waiting
+  for heavy task to finish before response.
```

Comment:
```suggestion
- [`fastify-delay-request`](https://github.com/climba03003/fastify-delay-request)
  Fastify plugin that queues requests that are sent whilst the server is starting
  or a high resource task is finishing.
```

Reply:
It is not clear enough since the server is already startup. But, the request is waiting for some jobs to finish before response.

Reply:
> It is not clear enough since the server is already startup and the request is waiting for some jobs to finish before response.

Cheers, that makes it clearer

Reply:
Have updated wording again.